api/lib/application/schooling-registration-dependent-users/index.js

```diff
@@ -37,6 +37,44 @@ exports.register = async function(server) {
         tags: ['api', 'schoolingRegistrationDependentUser']
       }
     },
+    {
+      method: 'POST',
+      path: '/api/schooling-registration-dependent-users/token',
```

Comment:
Proposition de @CHA 
```suggestion
      path: '/api/schooling-registration-dependent-users/external-user-token',
```
pour garder de la cohérence dans la base de code.

Reply:
Done